pkg/kustomize/secrets.go

```diff
@@ -406,9 +416,8 @@ func clairConfigFor(log logr.Logger, quay *v1.QuayRegistry, quayHostname, preSha
 			"migrations":             true,
 		},
 		"matcher": map[string]interface{}{
-			"connstring":    dbConn,
-			"max_conn_pool": 100,
```

Comment:
How do we decide what should be in this map and what should be in the connstring string?

Reply:
Clair's config is documented (via the go types) [here](https://pkg.go.dev/github.com/quay/clair/config#Config). In this specific instance, we [deprecated](https://pkg.go.dev/github.com/quay/clair/config#Matcher) the dedicated connection pool size to make it uniformly specified in the connection string.